template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/RazorPagesWeb-CSharp/Controllers/AccountController.cs

```diff
@@ -0,0 +1,115 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+#if (OrganizationalAuth || IndividualB2CAuth)
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authentication.Cookies;
+#if (IndividualB2CAuth)
```

Comment:
Nit: Probably easier to read the template conditions by having nested conditionals indented, something like this:
```
#if (OrganizationalAuth || IndividualB2CAuth) 
using Microsoft.AspNetCore.Authentication; 
using Microsoft.AspNetCore.Authentication.Cookies; 
    #if (IndividualB2CAuth) 
using Microsoft.AspNetCore.Authentication.Extensions; 
    #endif 
using Microsoft.AspNetCore.Authentication.OpenIdConnect; 
#endif
```

It does look a bit odd with the content indentation different from the conditional indentation, but makes it visually straightforward to find unbalanced conditionals.